ambry-cloud/src/main/java/com.github.ambry.cloud/HelixVcrCluster.java

```diff
@@ -99,7 +91,8 @@ public void addPartition(String partitionIdStr) {
         for (VirtualReplicatorClusterListener listener : listeners) {
           listener.onPartitionAdded(partitionId);
         }
-        logger.info("Partition {} is added to current VCR: {}", partitionIdStr, vcrInstanceName);
+        logger.info("Partition {} is added to current VCR: {}. Assigned partitions: {}", partitionIdStr,
+            vcrInstanceName, assignedPartitionIds);
```

Comment:
assignedPartitionIds can get large.  We may want to log the whole set only in debug mode, in case addPartition gets called many times on startup.